eclass/qmail.eclass

```diff
@@ -343,13 +345,14 @@ qmail_rootmail_fixup() {
 }
 
 qmail_tcprules_build() {
+	local f
 	for f in tcp.qmail-{smtp,qmtp,qmqp,pop3}; do
 		# please note that we don't check if it exists
 		# as we want it to make the cdb files anyway!
-		src="${ROOT}${TCPRULES_DIR}/${f}"
-		cdb="${ROOT}${TCPRULES_DIR}/${f}.cdb"
-		tmp="${ROOT}${TCPRULES_DIR}/.${f}.tmp"
-		[[ -e "${src}" ]] && tcprules "${cdb}" "${tmp}" < "${src}"
+		local src="${ROOT}${TCPRULES_DIR}/${f}"
+		local cdb="${ROOT}${TCPRULES_DIR}/${f}.cdb"
+		local tmp="${ROOT}${TCPRULES_DIR}/.${f}.tmp"
+		[[ -e ${src} ]] && tcprules "${cdb}" "${tmp}" < "${src}"
```

Comment:
Check exit status of `tcprules`?

Reply:
this whole eclass is already pretty brittle as is, I'd rather not lead to more unintended breakage